### ISSUE TITLE
Format or ignore the mixins `named parameters`

### ISSUE THREAD

**Issue by zoxon** [ThisIsManta/stylus-supremacy#26]
Hello, thank you for the wonderful tool.
I have a question about named mixin parameters.

Example:
This code is
```stylus
font-face(
	family: "Noto Sans",
	file-name: "regular",
	base-path: "../fonts/noto/sans",
	formats: eot woff2 woff truetype
)

.button_style_solid.button_color_gray
	button-style(
		color: $gray-6,
		background-color: $gray-2,
		background-color-hover: $gray-3
	)
```

formatted to this
```stylus
font-face(family: 'Noto Sans', file-name: 'regular', base-path: '../fonts/noto/sans', formats: eot woff2 woff truetype)
.button_style_solid.button_color_gray
  button-style(color: $gray-6, background-color: $gray-2, background-color-hover: $gray-3)
```

How can I disable formatting or better format more pretty (readable) like now?

**Comment by ThisIsManta** [comment]
Thank you for your opinion.

I agreed that it was hard to read when they were in the same line. Spare me sometime to find a solution for all.

**Comment by zoxon** [comment]
IMHO. The first thing to do is to disable formatting wiht inline comments, like in eslint.
https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments

**Comment by ThisIsManta** [comment]
Here's my solution:

If the original function call is written in a single line, for example `mixin(arg1: val1, arg2, val2)`, then the formatted content will be in a single line `mixin(arg1: val1, arg2, val2)`.

If the original function call has one or more arguments written in a different line, for example
```stylus
mixin(arg1: val1,
  arg2: val2)
```
then the formatted content will be distributed in different line
```stylus
mixin(
  arg1: val1,
  arg2: val2
)
```

This out-of-the-box solution has been implemented in version [version 2.10.0](https://github.com/ThisIsManta/stylus-supremacy/releases/tag/v2.10.0), which is available now. You don't have to config anything to get this work.

Personally, I don't like the way ESLint allow us to make an exception to some rules/files. I'd rather make a patch to fix a real issue.

Cheers!

**Comment by zoxon** [comment]
Thank You for the quick feedback
